### PR TITLE
upgraded project to use CMSIS-6

### PR DESCRIPTION
**Pull request by omarArm**
- Updated packs in csolution to latest versions
- Updated components accordingly
- Updated vcpkg-condfiguration file to use latest software versions of tools

### CODE REVIEW

File: hello.cproject.yml [hunk 1]
```diff
@@ -17,22 +17,23 @@ project:
...
   linker:
-    - script:  RTE/Device/STM32G474RETx/stm32g474.sct
+    - script: RTE/Device/STM32G474RETx/stm32g474.sct
       for-compiler: AC6
 
   components:
     - component: CMSIS:CORE
     - component: CMSIS:RTOS2:Keil RTX5&Source
 
-    - component: CMSIS-Compiler:I/O:STDOUT&User
```
Comment (KeilChris) [inline hello.cproject.yml:27]: I don't understand this change. You have removed the I/O component but changed the version of the CMSIS-Compiler pack to 2.0.0. If you don't use the component, you can remove that pack.

**Review by KeilChris — CHANGES_REQUESTED** [review]
Either ad the I/O component or remove the CMSIS-Compiler pack completely.

File: hello.cproject.yml (hunk 1 above)
Comment (omarArm) [reply hello.cproject.yml:27]: sorry my bad, I missed removing the pack as well

Comment (omarArm) [reply hello.cproject.yml:27]: Apologies for that, I have been meaning to erase the pack, but I missed it

**Review by KeilChris — APPROVED** [review]
LGTM

**Pull request merged**